### DISCUSSION TITLE
MagentaTV Telekom new IP

### DISCUSSION THREAD

**Discussion by Gortosch** [fabianishere/udm-iptv#250] [Q&A]
Hello,

the proxy runs perfectly without any problems. 
The only problem I have is that when the provider (Deutsche Telekom) assigns me a new IP, the image is frozen and the following error message appears in the dignose log:

_root@Firewall:~# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: false (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15
LAN Interfaces: br60
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
51: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 84.132.XX.XX peer 62.155.243.150/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.243.150 proto kernel scope link src 84.132.XX.XX
=== Service Logs ===
Apr 07 20:04:32 Firewall igmpproxy[967622]: select() failure; Errno(4): Interrupted system call
Apr 07 20:04:32 Firewall igmpproxy[967622]: can't leave group 232.0.10.149 on interface ppp0; Errno(99): Cannot assign requested address
Apr 07 20:04:32 Firewall systemd[1]: udm-iptv.service: Succeeded.
Apr 07 20:04:32 Firewall systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Apr 07 20:04:32 Firewall systemd[1]: udm-iptv.service: Consumed 15.684s CPU time.
Apr 07 20:04:32 Firewall systemd[1]: Started IPTV support for the UniFi Dream Machine.
Apr 07 20:04:32 Firewall udm-iptv[3569625]: NATing IPTV network ranges (if necessary)
Apr 07 20:04:32 Firewall udm-iptv[3569625]: Monitoring interface ppp0 for changes
Apr 07 20:04:32 Firewall udm-iptv[3569625]: Setting up igmpproxy
Apr 07 20:04:32 Firewall udm-iptv[3569625]: Starting igmpproxy_

If I then restart the service, everything runs smoothly again until a new IP change is carried out.

_root@Firewall:~# systemctl restart udm-iptv
root@Firewall:~# udm-iptv-diag
Please share the following output with the developers:
=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: false (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15
LAN Interfaces: br60
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
51: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
    inet 84.132.XX.XX peer 62.155.243.150/32 scope global ppp0
       valid_lft forever preferred_lft forever
62.155.243.150 proto kernel scope link src 84.132.XX.XX
=== Service Logs ===
Apr 17 17:04:00 Firewall systemd[1]: Stopping IPTV support for the UniFi Dream Machine...
Apr 17 17:04:00 Firewall igmpproxy[3569625]: select() failure; Errno(4): Interrupted system call
Apr 17 17:04:00 Firewall systemd[1]: udm-iptv.service: Succeeded.
Apr 17 17:04:00 Firewall systemd[1]: Stopped IPTV support for the UniFi Dream Machine.
Apr 17 17:04:00 Firewall udm-iptv[925247]: NATing IPTV network ranges (if necessary)
Apr 17 17:04:00 Firewall systemd[1]: udm-iptv.service: Consumed 22.842s CPU time.
Apr 17 17:04:00 Firewall systemd[1]: Started IPTV support for the UniFi Dream Machine.
Apr 17 17:04:00 Firewall udm-iptv[925247]: Monitoring interface ppp0 for changes
Apr 17 17:04:00 Firewall udm-iptv[925247]: Setting up igmpproxy
Apr 17 17:04:00 Firewall udm-iptv[925247]: Starting igmpproxy_

Have I configured something incorrectly?

Best regards 
Sven

**Comment by frgtv10** [comment]
did you get any updates on this?

**Reply by Gortosch** [reply]
Hi,

I had noticed that I had a very old version. After an update, there was another IP change. It worked perfectly then. Since then, I have not been assigned a new IP. However, I am about to update to 7.4.156. In this update, IGMP Proxy is supposed to work in the interface. So far, however, I have not dared to do it, as everything is working fine.

Greetings
Sven

**Comment by DrLapacho** [comment]
Hi There,

Switched from Telekom-DSL to Telekom-FTTH today
Now MagentaTV stucks after 7-10 seconds

**Diagnostic Information**
=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 0 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
27: ppp0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1492 qdisc pfifo_fast state UNKNOWN group default qlen 3
inet 84.145.154.15 peer 62.155.240.95/32 scope global ppp0
valid_lft forever preferred_lft forever
62.155.240.95 proto kernel scope link src 84.145.154.15
=== Service Logs ===
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@208]: if_mtu = 9216
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@214]: if_falgs = 4163
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@178]: ppp0's ipv4 address is 84.145.154.15
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@195]: if_index = 27
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@208]: if_mtu = 1492
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: DEBUG[init_interface@214]: if_falgs = 4305
Jan 19 21:32:00 UDM-Pro udm-iptvd[12335]: ERROR[main@619]: exiting.........
Jan 19 21:32:00 UDM-Pro IMPROXY[12335]: ERROR[main@619]: exiting.........
Jan 19 21:32:00 UDM-Pro systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Jan 19 21:32:00 UDM-Pro systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

===ifconfig==
root@UDM-Pro:~# ifconfig
br0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 9216
inet 192.168.2.1 netmask 255.255.255.0 broadcast 0.0.0.0
inet6 fe80::e438:83ff:fe23:52fa prefixlen 64 scopeid 0x20
ether e6:38:83:23:52:fa txqueuelen 1000 (Ethernet)
RX packets 28802 bytes 6851889 (6.5 MiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 80713 bytes 94754358 (90.3 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52f9 prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:f9 txqueuelen 1000 (Ethernet)
RX packets 12 bytes 1154 (1.1 KiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 20 bytes 3906 (3.8 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52fa prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:fa txqueuelen 1000 (Ethernet)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 17 bytes 3016 (2.9 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52fb prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:fb txqueuelen 1000 (Ethernet)
RX packets 7 bytes 338 (338.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 22 bytes 4126 (4.0 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52fc prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:fc txqueuelen 1000 (Ethernet)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 18 bytes 3166 (3.0 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth4: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52fd prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:fd txqueuelen 1000 (Ethernet)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 18 bytes 3166 (3.0 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52fe prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:fe txqueuelen 1000 (Ethernet)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 18 bytes 3166 (3.0 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth6: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:52ff prefixlen 64 scopeid 0x20
ether e4:38:83:23:52:ff txqueuelen 1000 (Ethernet)
RX packets 47 bytes 3608 (3.5 KiB)
RX errors 0 dropped 3 overruns 0 frame 0
TX packets 19 bytes 3252 (3.1 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth7: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:5300 prefixlen 64 scopeid 0x20
ether e4:38:83:23:53:00 txqueuelen 1000 (Ethernet)
RX packets 37 bytes 3680 (3.5 KiB)
RX errors 0 dropped 3 overruns 0 frame 0
TX packets 20 bytes 3362 (3.2 KiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth8: flags=4099<UP,BROADCAST,MULTICAST> mtu 9216
inet6 fe80::e638:83ff:fe23:5301 prefixlen 64 scopeid 0x20
ether e4:38:83:23:53:01 txqueuelen 1000 (Ethernet)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 3 bytes 304 (304.0 B)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth9: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:5302 prefixlen 64 scopeid 0x20
ether e4:38:83:23:53:02 txqueuelen 10000 (Ethernet)
RX packets 69164 bytes 81130682 (77.3 MiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 18032 bytes 5486086 (5.2 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth10: flags=4099<UP,BROADCAST,MULTICAST> mtu 1500
inet 192.168.1.2 netmask 255.255.255.0 broadcast 0.0.0.0
ether e4:38:83:23:53:03 txqueuelen 10000 (Ethernet)
RX packets 0 bytes 0 (0.0 B)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 0 bytes 0 (0.0 B)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

eth9.7: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 1500
inet6 fe80::e638:83ff:fe23:5302 prefixlen 64 scopeid 0x20
ether e4:38:83:23:53:02 txqueuelen 1000 (Ethernet)
RX packets 69164 bytes 81130682 (77.3 MiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 18034 bytes 5736219 (5.4 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING> mtu 65536
inet 127.0.0.1 netmask 255.0.0.0
inet6 ::1 prefixlen 128 scopeid 0x10
loop txqueuelen 1000 (Local Loopback)
RX packets 139285 bytes 41839826 (39.9 MiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 139285 bytes 41839826 (39.9 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

ppp0: flags=4305<UP,POINTOPOINT,RUNNING,NOARP,MULTICAST> mtu 1492
inet 84.145.154.15 netmask 255.255.255.255 destination 62.155.240.95
ppp txqueuelen 3 (Point-to-Point Protocol)
RX packets 69013 bytes 80566065 (76.8 MiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 17988 bytes 5338272 (5.0 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

switch0: flags=4419<UP,BROADCAST,RUNNING,PROMISC,MULTICAST> mtu 9216
inet6 fe80::e438:83ff:fe23:52fa prefixlen 64 scopeid 0x20
ether e6:38:83:23:52:fa txqueuelen 1000 (Ethernet)
RX packets 29704 bytes 6906145 (6.5 MiB)
RX errors 0 dropped 0 overruns 0 frame 0
TX packets 80924 bytes 93666382 (89.3 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

switch0.1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST> mtu 9216
inet6 fe80::e438:83ff:fe23:52fa prefixlen 64 scopeid 0x20
ether e6:38:83:23:52:fa txqueuelen 1000 (Ethernet)
RX packets 28826 bytes 6853541 (6.5 MiB)
RX errors 0 dropped 3 overruns 0 frame 0
TX packets 80713 bytes 94754358 (90.3 MiB)
TX errors 0 dropped 0 overruns 0 carrier 0 collisions 0

**Describe the Bug**
After switsching from DSL to FTTH today Magenta TV stucks after 7-10 seconds.
All worked fine with DSL
UDM-Pro v3.2.9; Network V8.0.28

![image](https://github.com/fabianishere/udm-iptv/assets/139555237/4abce1a4-5690-416b-95d7-210c771cf83e)
![image](https://github.com/fabianishere/udm-iptv/assets/139555237/390a438f-f231-408e-a242-8e43d18cca12)
![image](https://github.com/fabianishere/udm-iptv/assets/139555237/5b3eb8fa-a646-4017-a162-58680650dbfd)


**udm-iptv.conf**

IPTV_WAN_INTERFACE="ppp0"
IPTV_WAN_VLAN="0"
IPTV_WAN_VLAN_INTERFACE="iptv"
IPTV_WAN_RANGES="224.0.0.0/4 87.141.0.0/16 193.158.0.0/15"
IPTV_WAN_DHCP_OPTIONS="-O staticroutes -V IPTV_RG"
IPTV_LAN_INTERFACES="br0"
IPTV_IGMPPROXY_PROGRAM="improxy"
IPTV_IGMPPROXY_DISABLE_QUICKLEAVE="false"
IPTV_IGMPPROXY_DEBUG="true"
IPTV_IGMPPROXY_IGMP_VERSION="3"

My mother-in-law can't watch TV, I suffer ;)
Thanks for help.
Michael

**Comment by dmutsaers** [comment]
I think you have to disable IGMP Proxy in the UniFi Network Application if you install and use udm-iptv.

**Comment by Gortosch** [comment]
Hi,

oh man I'm also getting fiber optics from Telekom next week. I am currently using the IGMP proxy in the UniFi Network Application. This is now working perfectly. I hope I won't have any problems after the switch. Has anyone already successfully made this switch? A lot of connections are currently being converted to fiber optics.

@DrLapacho Have you been able to solve the problem in the meantime?

Regards
Sven

**Reply by DrLapacho** [reply]
i solved the problem by switching to MagentaTV One boxes and sending the MR401/201 back to Telekom.

**Comment by Gortosch** [comment]
Hmmm,

I have the MR401.... and keep my fingers crossed...
